docs/azure_arc_jumpstart/azure_arc_k8s/alibaba/alibaba_terraform/_index.md

```diff
@@ -0,0 +1,201 @@
+  export ALICLOUD_PROFILE="<Aliyun CLI profile>"
+  export AZURE_SUBSCRIPTION_ID="<Your Azure subscription id>"
+  export AZURE_LOCATION="<Azure location>"
+  export AZURE_RESOURCE_GROUP="<Azure resource group>"
+  export AZURE_CLUSTER_NAME="<Azure Arc cluster name>"
+  ```
+
+  For example:
+
+  ```shell
+  export ALICLOUD_ACCESS_KEY="aBcDeFgHiJkLmNo"
+  export ALICLOUD_SECRET_KEY="aBcDeFgHiJkLmNoPqRtUvWxYz"
+  export ALICLOUD_REGION="cn-shanghai"
+  export ALICLOUD_PROFILE="akProfile"
+  export AZURE_SUBSCRIPTION_ID="XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
+  export AZURE_LOCATION="southeastasia"
+  export AZURE_RESOURCE_GROUP="Azure-Arc-Aliyun-K8S"
+  export AZURE_CLUSTER_NAME="azure-arc-aliyun"
+  ```
+
+  > **Note: If you are running in a PowerShell environment, to set the Terraform environment variables, use the _$env:variable-name=variable-value_ (see example below)**
+
+  ```powershell
+  $env:ALICLOUD_ACCESS_KEY="aBcDeFgHiJkLmNo"
+  ```
+
+* From folder `azure_arc_k8s_jumpstart/alibaba/terraform` run the ```terraform init``` command which will download the required terraform providers.
+
+  ![terraform init output](./05.png)
+
+* Deploy cluster by running the ```terraform apply --auto-approve``` command. Wait for the plan to finish.
+
+  ![terraform deploy output](./06.png)
+  ...
+  ![terraform deploy output](./07.png)
+
+  sample cluster in Alibaba Cloud Resource Management view:
+
+  ![Alibaba Cloud Resource Management](./08.png)
+
+* The plan will drop the created clusters configuration in users home directory `~/.kube/config` to be used with `kubectl` or `helm`. But to make it usable with `helm`, file permissions need to be reduced:
+
+   ```shell
+   chmod go-r ~/.kube/config
+   ```
+
+## Connecting to Azure Arc
+
+* Now that you have running cluster, based on the environment variables set and Azure environment prepared above, you can connect the Alibaba Cloud cluster to Azure Arc:
+
+   ```shell
+   az login
+   az account set -s $AZURE_SUBSCRIPTION_ID
+   az group create --location $AZURE_LOCATION --name $AZURE_RESOURCE_GROUP
+   ```
+
+   output should look like this:
+
+   ```json
+   {
+     "id": "/subscriptions/XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX/resourceGroups/Azure-Arc-Aliyun-K8S",
+     "location": "southeastasia",
+     "managedBy": null,
+     "name": "Azure-Arc-Aliyun-K8S",
+     "properties": {
+       "provisioningState": "Succeeded"
+     },
+     "type": "Microsoft.Resources/resourceGroups"
+   }
+   ```
+
+   connect the cluster:
+
+   ```shell
+   az connectedk8s connect --name $AZURE_CLUSTER_NAME --resource-group $AZURE_RESOURCE_GROUP --location $AZURE_LOCATION
+   ```
+
+   sample output
+
+   ![Connect Alibaba cluster to Azure Arc](./09.png)
+
+   sample cluster in Azure Portal
+
+   ![Connect Alibaba cluster to Azure Arc](./10.png)
+
+## Delete the deployment
+
+To delete the environment in Azure, the use *`az group delete --resource-group $AZURE_RESOURCE_GROUP -y`* command.
```

Comment:
Add output screenshot as well as a portal screenshot on deleting the resource group